### ISSUE TITLE
[Lang] Python Snippets

### ISSUE THREAD

**Issue by abishekk92** [solana-dev-adv/solana-cookbook#133]
It would be useful to have RPC client-side snippets in python

**Comment by saurabhghatnekar** [comment]
Hey, @abishekk92 I'd like to contribute. Can you please elaborate on this issue/ what needs to be done?

**Comment by abishekk92** [comment]
Yes, that's precisely the idea. IMO, existing recipes would be a good place to start. Perhaps, @jacobcreech can chime if this is in line with he has in mind for rust. See #127

**Comment by jacobcreech** [comment]
@abishekk92 That is precisely what the issue is for, creating snippets I'm each language

@saurabhghatnekar You are correct. If you'd like to start, just name which part you're going after here so people wanting to do more don't duplicate your work

**Comment by saurabhghatnekar** [comment]
Cool. I'll take up the NFT recipe. Will add  a folder 'python' in 'nfts' folder

**Comment by jacobcreech** [comment]
@saurabhghatnekar in the code snippet's folder you can add your `.py` files alongside the others

**Comment by saurabhghatnekar** [comment]
Alright
